torch/_torch_docs.py

```diff
@@ -4431,6 +4431,26 @@ def merge_dicts(*dicts):
 """)
 
 
+add_docstr(torch.result_type,
+           r"""
+result_type(tensor, other) -> ScalarType
+
+Returns the :class:`torch.dtype` that would result from performing an arithmetic
+operation on the provided input tensors.
+
+Args:
+    tensor (Tensor): an input tensor
```

Comment:
can both arguments not be Tensors or Number (or scalar, I don't know how we do this consistently in documentation).

Also, can you link this to the explanation of type promotion we have elsewhere in the docs?

Reply:
it's in the linked torch.dtype doc but added an explicit reference.